<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChannelSettings</class>
 <widget class="QWidget" name="ChannelSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="channelLayout">
     <item>
      <widget class="QLabel" name="channelName">
       <property name="text">
        <string>Channel name</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="inputLayout">
     <item>
      <widget class="QLabel" name="inputLabel">
       <property name="text">
        <string>Input</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="inputBox"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="filterLayout">
     <item>
      <widget class="QLabel" name="filterLabel">
       <property name="text">
        <string>Filter</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="filterBox"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="wrapLayout">
     <item>
      <widget class="QLabel" name="wrapLabel">
       <property name="text">
        <string>Wrap</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="wrapBox"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
